scripts/.air.toml

```diff
@@ -10,7 +10,7 @@ tmp_dir = ".air"
   --tags "" \
   -o ./.air/bytebase ./backend/bin/server/main.go"""
   delay = 500
-  exclude_dir = [".air", "vendor", "frontend", "docs"]
+  exclude_dir = [".air", "bytebase-build", "docs", "frontend", "helm-charts", "proto", "quickstart", "scripts",  "vendor"]
   exclude_file = []
```

Comment:
How about add *_test.go

Reply:
sync'ed offline that we will skip testing for now. I've done some manual testing myself.